mechanics.opy

```diff
@@ -94,6 +94,7 @@ rule "Checking | In the air":
     @Condition eventPlayer.isInAir() 
     eventPlayer.JumpCount = 1
 
+/* Remove this rule? */
```

Comment:
Does it need to be removed? Looks like it assists with ban emote. Or does your new detection cover this?